montecarlo/cfast_mcarlo.py

```diff
@@ -835,7 +836,34 @@ def _draw_targets(self):
                                 'DEPTH_UNITS': 'M'
                                 })
         if targets:
-            self.sections['DEVC'] = targets
+            self.sections.setdefault('DEVC', []).extend(targets)
+    def _draw_window_targets(self):
+        targets = []
+        for v in self.s.query("SELECT v.name, v.vent_from_name, v.face, v.face_offset, v.width as wwidth, v.depth as wdepth, v.sill, v.height, r.width, r.depth FROM aamks_geom v JOIN aamks_geom r on v.vent_from_name = r.name WHERE v.type_sec='WIN' AND v.how_much_open=0"):
+            z = round((v['sill']+v['height']*0.5)/100, 2)
+            if v['face'] == 'RIGHT':
+                x = 0
+                y = round((v['depth']-v['face_offset']-v['wdepth']*0.5)/100, 2)
+            if v['face'] == 'LEFT':
+                x = v['width']/100
+                y = round((v['face_offset']+v['wdepth']*0.5)/100, 2)
+            if v['face'] == 'REAR':
+                x = round((v['width']-v['face_offset']-v['wwidth']*0.5)/100, 2)
+                y = v['depth']/100
+            if v['face'] == 'FRONT':
+                x = round((v['face_offset']+v['wwidth']*0.5)/100, 2)
+                y = 0
+            targets.append({'ID': f"t_{v['name']}",
+                            'COMP_ID': v['vent_from_name'],
+                            'LOCATION': [x, y, z],
+                            'TYPE': 'PLATE',
+                            'MATL_ID': '',
+                            'SURFACE_ORIENTATION': 'CEILING',
```

Comment:
Should be one of walls, shouldn't it?

I'm not sure, but for plate `DEVC` there should be also a `NORMAL` defined.

Reply:
Changed  in 4e10417